setup.py

```diff
@@ -1,10 +1,13 @@
+import os
 import sys
 from pathlib import Path
 
 import cmake_build_extension
 import setuptools
 
 
+SETUPTOOLS_CMAKE_OPTIONS = os.environ.get("SETUPTOOLS_CMAKE_OPTIONS", "-DBUILD_TESTS=OFF").split()
```

Comment:
What's using this?

Reply:
I was using this to test whether inserting `GOOGLETEST_ROOT` into the cmake options would solve the segfault; I can remove it if desired

Reply:
Yeah, just revert this file unless you really need the changes.